File: jpa_2023_c3_e1/src/main/java/org/example/entities/Employee.java
package org.example.entities;

import jakarta.persistence.*;

import java.util.UUID;

@Entity
@Table(name = "employees")
public class Employee {

    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    @Column(name = "id")
    private String id;

    private String name;
    private String address;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    @Override
    public String toString() {
        return "Employee{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", address='" + address + '\'' +
                '}';
    }
}

File: gpt_e2_e1/src/main/kotlin/com/example/gpt_e2_e1/model/Employee.kt
package com.example.gpt_e2_e1.model

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table
import java.util.*

@Table("employees")
data class Employee(
        @Id val id: Long? = null,
        val name: String,
        val jobTitle: String,
        val department: String,
        val internalId: UUID
)
